src/components/charts/modal/CreateChartGroup.tsx

```diff
@@ -27,19 +27,50 @@ export default class CreateChartGroup extends Component<CreateChartGroupProps, C
     }
 
     handleNameChange(event) {
-        this.setState({ name: {value: event.target.value, error: ""} });
+        this.setState({ name: { value: event.target.value, error: [] } });
     }
 
     handleDescriptionChange(event) {
         this.setState({ description: event.target.value });
     }
 
     async saveChartGroup(e) {
-        const nameRegexp = new RegExp(`^[a-z]+[a-z0-9\-\?]*[a-z0-9]+$`)
-        if (!nameRegexp.test(this.state.name.value)) {
-            this.setState({ name: { ...this.state.name, error: 'name must follow `^[a-z]+[a-z0-9\-\?]*[a-z0-9]+$` pattern.'}})
-            return
+        const lowercaseRegexp = new RegExp('^[a-z0-9-. ][a-z0-9-. ]*[a-zA-Z0-9-. ]$')
```

Comment:
i think this regex is wrong 
'^[a-z]+[a-z0-9\-\. ]*[a-z0-9]$'

Reply:
`'^[a-z]+[a-z0-9-. ]*[a-z0-9]$'` this regex was fine if we have  to show error in a single line